### PR TITLE
[pt-br] Add docs/tasks/debug/debug-application/debug-init-containers.md

### PR DESCRIPTION
**Pull request by AnaCarolinaRodriguesLeite**
Hello everyone, 
Brazilian portuguese translation for the [Debug Init Containers Page](https://github.com/kubernetes/website/blob/main/content/en/docs/tasks/debug/debug-application/debug-init-containers.md).

### CODE REVIEW

**Comment by sftim** [comment]
/retitle Localize “Debug Init Containers” to Portuguese

File: content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+---
+reviewers:
+- bprashanth
+- enisoc
+- erictune
+- foxish
+- janetkuo
+- kow3ns
+- smarterclayton
```
Comment (sftim) [inline content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md:9]:
```suggestion
```
Those people probably don't speak Brazilian Portuguese.

File: content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+title: Debug Init Containers
```
Comment (sftim) [inline content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md:10]: Please localize this title.

File: content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+```shell
+kubectl logs <pod-name> -c <init-container-2>
+```
+
+Inicializações de contêineres que executam uma impressão de comandos de script de shell
+à medida que são executados. Por exemplo, você pode fazer isso no Bash executando `set -x` no início do script.
+
+<!-- discussion -->
+
+## Entendendo sobre o status do pod
```
Comment (sftim) [inline content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md:113]: To make the internal link work:
```suggestion
## Entendendo sobre o status do pod {#understanding-pod-status}
```

File: content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+Um status do Pod começando com `Init:` resume o status da execução da inicialização de contêineres. 
+A tabela abaixo descreve alguns valores de status de exemplo que você pode ver durante a depuração de Init Containers.
```
Comment (sftim) [inline content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md:116]:
```suggestion
A tabela abaixo descreve alguns valores de status de exemplo que você pode ver durante a depuração de contêineres de inicialização.
```
?

File: content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+- smarterclayton
+title: Debug Init Containers
+content_type: task
+weight: 40
+---
+
+<!-- overview -->
+
+Esta página mostra como investigar problemas relacionados à execução de
+inicialização de Contêineres. As linhas de comando de exemplo abaixo
```
Comment (sftim) [inline content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md:18]:
```suggestion
contêineres de inicialização. As linhas de comando de exemplo abaixo
```

File: content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+
+```shell
+kubectl get pod nginx --template '{{.status.initContainerStatuses}}'
+```
+
+Este comando retornará as mesmas informações acima em JSON bruto.
+
+## Acessando logs de inicialização de contêineres
+
+Passe o nome da inicialização de contêineres junto com o nome do Pod
```
Comment (sftim) [inline content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md:101]:
```suggestion
Passe o nome da contêiner de inicialização junto com o nome do Pod
```

File: content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+Você também pode acessar programaticamente os status das inicializações de contêineres,
+lendo o campo `status.initContainerStatuses` nas especificações do pod:
+
+```shell
+kubectl get pod nginx --template '{{.status.initContainerStatuses}}'
+```
+
+Este comando retornará as mesmas informações acima em JSON bruto.
+
+## Acessando logs de inicialização de contêineres
```
Comment (sftim) [inline content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md:99]:
```suggestion
## Acessando logs de contêineres de inicialização
```

File: content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+      Reason:        Error
+      Exit Code:     1
+      Started:       ...
+      Finished:      ...
+    Ready:           False
+    Restart Count:   3
+    ...
+```
+
+Você também pode acessar programaticamente os status das inicializações de contêineres,
```
Comment (sftim) [inline content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md:90]:
```suggestion
Você também pode acessar programaticamente os status das contêineres de inicialização
```

File: content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+
+```
+NAME         READY     STATUS     RESTARTS   AGE
+<pod-name>   0/1       Init:1/2   0          7s
+```
+
+Consulte [Entendendo sobre o status do pod](#understanding-pod-status) para obter mais exemplos de
+valores de status e seus significados.
+
+## Obtendo detalhes sobre a Inicialização de Contêineres
```
Comment (sftim) [inline content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md:52]:
```suggestion
## Obtendo detalhes sobre a contêineres de inicialização
```

File: content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+referem-se ao pod como `<pod-name>` e a inicialização de contêineres como `<init-container-1>` e
+`<init-container-2>`.
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+* Você deve estar familiarizado com os fundamentos da
+  [inicialização de Contêineres](/docs/concepts/workloads/pods/init-containers/).
```
Comment (sftim) [inline content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md:28]:
```suggestion
  [contêineres de inicialização](/docs/concepts/workloads/pods/init-containers/).
```

File: content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md [hunk 11]
```diff
@@ -0,0 +1,124 @@
...
+* Você deve ter [Configurado uma Inicialização de Contêiner](/docs/tasks/configure-pod-container/configure-pod-initialization/#creating-a-pod-that-has-an-init-container/).
```
Comment (sftim) [inline content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md:29]:
```suggestion
* Você deve ter [configurado uma contêiner de inicialização](/docs/tasks/configure-pod-container/configure-pod-initialization/#creating-a-pod-that-has-an-init-container/).
* ```

File: content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md [hunk 12]
```diff
@@ -0,0 +1,124 @@
...
+
+<!-- steps -->
+
+## Verificando o status das inicializações de contêineres
```
Comment (sftim) [inline content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md:33]:
```suggestion
## Verificando o status das contêineres de inicializações
```

**Review by sftim — COMMENTED** [review]
“init containers” are _containers for performing initialization_ (of a Pod). That is different from (the process of) initializing a container.

I don't speak Portuguese at all well, but I have concerns that this PR might need substantial changes. I've added partial inline feedback towards that outcome.

If I'm wrong, please comment accordingly and I can revise my review.

**Comment by stormqueen1990** [comment]
/retitle [pt-br] Add docs/tasks/debug/debug-application/debug-init-containers.md

File: content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md (hunk 1 above)
Comment (stormqueen1990) [reply content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md:9]: This section can be fully removed.

File: content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md [hunk 13]
```diff
@@ -0,0 +1,124 @@
...
+title: Debug Init Containers
+content_type: task
+weight: 40
+---
+
+<!-- overview -->
+
+Esta página mostra como investigar problemas relacionados à execução de
+inicialização de Contêineres. As linhas de comando de exemplo abaixo 
+referem-se ao pod como `<pod-name>` e a inicialização de contêineres como `<init-container-1>` e
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md:19]: "e a inicialização de contêineres" should be "e aos contêineres de inicialização".

As @sftim pointed out, "Init Containers" refer to a specific kind of container in Kubernetes tasked with initializing a pod.

File: content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md (hunk 12 above)
Comment (stormqueen1990) [reply content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md:33]: I think this suggestion mostly works, except that "inicializações" should be singular.

**Review by stormqueen1990 — COMMENTED** [review]
Hello there, @AnaCarolinaRodriguesLeite! 👋 

Thank you very much for your contribution.

As @sftim pointed out, there are quite a few places where the wording has to be updated to match what an init container is. Could you please review these points and address them?

Thanks again, and please don't hesitate to reach out on Slack if you need any help.

File: content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md [hunk 14]
```diff
@@ -0,0 +1,115 @@
+---
+title: Contêineres de inicialização de depuração
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md:2]: This translation is inaccurate. The page talks about how to debug init containers; therefore, this should be "Depurando Contêineres de Inicialização".

File: content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md [hunk 15]
```diff
@@ -0,0 +1,115 @@
...
+
+## Obtendo detalhes sobre os contêineres de inicialização
+
+Veja informações mais detalhadas sobre a execução da Inicialização de Contêineres:
+
+```shell
+kubectl describe pod <pod-name>
+```
+
+Por exemplo, um pod com duas inicializações de contêineres pode mostrar o seguinte:
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md:51]: In these sentences:
* "inicialização de contêineres" should be "contêineres de inicialização".
* No need to capitalize those words.

File: content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md [hunk 16]
```diff
@@ -0,0 +1,115 @@
...
+## Acessando logs de contêineres de inicialização
+
+Passe o nome do contêiner de inicialização junto com o nome do Pod
+para acessar seus logs.
+
+```shell
+kubectl logs <pod-name> -c <init-container-2>
+```
+
+Inicializações de contêineres que executam uma impressão de comandos de script de shell
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md:99]: Some bits here:
* "inicialização de contêineres" should be "contêineres de inicialização".
* The bit about printing shell commands is not super accurate. The meaning here is that you'll see output when an init container runs shell scripts. I suggest using "contêineres de inicialização que executam comandos de script de shell imprimem ...".

File: content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md [hunk 17]
```diff
@@ -0,0 +1,115 @@
...
+à medida que são executados. Por exemplo, você pode fazer isso no Bash executando `set -x` no início do script.
+
+<!-- discussion -->
+
+## Entendendo sobre o status do pod { #understanding-pod-status }
+
+Um status do Pod começando com `Init:` resume o status da execução da inicialização de contêineres.
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md:106]: "inicialização de contêineres" should be "contêineres de inicialização".

File: content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md [hunk 18]
```diff
@@ -0,0 +1,115 @@
...
+Um status do Pod começando com `Init:` resume o status da execução da inicialização de contêineres. 
+A tabela abaixo descreve alguns valores de status de exemplo que você pode ver durante a depuração de contêineres de inicialização.
+
+Status | Significado
+------ | -------
+`Init:N/M` | O pod tem inicializações de contêineres `M` e `N` que foram concluídas até agora.
+`Init:Error` | Uma inicialização de contêiner falhou ao executar.
+`Init:CrashLoopBackOff` | Uma inicialização de contêiner falhou repetidamente.
+`Pending` | O pod ainda não começou a executar a inicialização de contêineres.
+`PodInitializing` ou `Running` | O pod já concluiu a execução das inicializações de contêineres.
```
Comment (stormqueen1990) [inline content/pt-br/docs/tasks/debug/debug-application/debug-init-containers.md:115]: In all of those sentences "inicialização de contêineres" should be "contêineres de inicialização".

**Review by stormqueen1990 — COMMENTED** [review]
Hello there, @AnaCarolinaRodriguesLeite! 👋 

Thank you very much for your contribution! 🎉 

Overall, the text is good in this translation, but there are still many occurrences of "inicialização de contêineres" that should have been "contêineres de inicialização". The title is also a bit weird, I made a suggestion but feel free to use something else, as long as it delivers the meaning of debugging an init container.

If you have any questions or need help, please feel free to reach out to us on the `#kubernetes-docs-pt` channel of Kubernetes Slack. 😃

**Pull request merged**